apps/02RockPaperScissors0.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+
+var assert = require('assert');
+var prompt = require('prompt');
+// this is what starts the app
+prompt.start();
+
+// this is where the code test one hand verse another hand.
+function rockPaperScissors(hand1, hand2) {
+     hand1 = hand1.toLowerCase();
+     hand2 = hand2.toLowerCase();
+
+    if (hand1===hand2) {
+      return "It's a tie!";
+    };
+
+    if ((hand1 === 'rock') && (hand2 === 'scissors')){
+      return "Hand one wins!";
+    }
+    else if ((hand1 === 'paper') && (hand2 ==='rock')){
+      return 'Hand one wins!';
+    }
+    else if ((hand1 === 'scissors')&& (hand2 ==='paper')){
+      return 'Hand one wins!';
+    }
+    else if ((hand1 === 'rock')&& (hand2 ==='paper')){
+      return 'hand two wins!';
+    }
+    else if ((hand1 === 'paper') && (hand2 ==='scissors')){
+      return 'Hand two wins!';
+    }
+    else if ((hand1 === 'scissors')&& (hand2==='rock')){
+      return 'Hand two wins!';
+    }
+    else {
+      return 'please enter rock, paper, or scissors';
+    };
```

Comment:
Sound logic here. Good job accounting for invalid answers.

![game](http://img.photobucket.com/albums/v161/stellkins/edward_scissorhands-1.gif)